src/components/Feed/Post.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+export default function Post({
+	user = { avatar: "", name: "Mike", tag: "@mikey", time: "1h" },
```

Comment:
If you extract this default object out into a variable, you'll be able to fit this function signature neatly on one line